src/main/resources/templates/experiment_page_create_experiment.html

```diff
@@ -13,10 +13,7 @@
                 <p>&nbsp;</p>
                 <h2 class="team-heading team-heading-current-team" th:text="#{exp.createExp}">Create Experiment</h2>
 				<div th:if="${message}">
-					<h4 th:text="${message}"></h4>
-				</div>
-				<div th:if="${message2}">
-					<h4 th:text="${message2}"></h4>
+					<p><i class="fa fa-exclamation-circle btn-fa-red"></i> <span class="account-change-fail" th:text="${message}"></span></p>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)